modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptFetchClientCodegen.java

```diff
@@ -916,6 +939,30 @@ private void setPrefixParameterInterfaces(boolean prefixParameterInterfaces) {
         public String dataTypeAlternate;
         public boolean isUniqueId; // this parameter represents a unique id (x-isUniqueId: true)
 
+        public boolean itemsAreUniqueId() {
```

Comment:
mostly a code refactoring to improve and unify between similar codegen property and operation logic. Includes some edge cases fixes and improvements to typings.